### PR TITLE
UI updates

### LINKED ISSUE #1235: As a Admin Level User (Bonnie), I want to know be able to tell what each tab on the navigation tab means, so I don't have to guess each time.
### Overview
The existing navigation bar currently has 4 tabs: 1) Home, 2) Events, 3) Admin, 4) Projects. It is not very clear on what each tab provides for our users. To improve usability and simplify navigation, we need to rethink the nav bar.

The Home tab currently displays a "donut report" that is not very valuable to our user. This information under Home tab is not being actively used and is taking up valuable real estate on the nav bar.
The Events tab leads to a search function. While this feature is somewhat valuable, in its current form, it's not usable because it's taking too long to load. We will hide the Events tab for now.
The Admin tab leads to a page with two links: a) User Management, b) Projects. Clicking on User Management leads to page with an email search function, to then add a volunteers to a project. The Projects link is redundant as there is a separate tab just for Projects. To simplify our nav bar, we should rename the Admin tab to User (Management) tab that takes user directly to email search function.
Projects tabs is clear and functional. This should remain

### Action Items
- [ ] Hide Home Tab
- [ ] Hide Events Tab
- [ ] Rename Admin to User (Management) tab
- [ ] New User (Management) tab should lead directly to email search function

### Acceptance Criteria
Gherkin Syntax:
    Scenarios: All the actions a user could take (including bad input)
    GIVEN: Sets the context. What page are we on and what state are we in? Is the user an admin? Signed-in? Has created a campaign?
    WHEN: What actions the user is performing. What event occurred?
    THEN: What should the system do in response? What is the expected outcome?

Scenario: Admin level logs into VRMS
GIVEN: That the user has Admin level access
WHEN they log on to their VRMS account
THEN the navigation bar should have a) User (Management) tab and b) Projects tab
AND User (Management) tab should lead directly to emails search function

### Resources/Instructions
Zoom time stamp [20:57]
Zoom recording - https://drive.google.com/drive/u/0/folders/1oSUAaU-fzKHpXeZpou5PH3SwC5eRnEWq
Screen Shot 2022-11-02 at 5 38 27 PM

### PR DESCRIPTION
**Pull request by jbubar**
Fixes (or is a great start for) #1235

Additional changes were made prior to having more development team members. If we can use these changes, great. Otherwise, we can cherry pick out the parts that that pertain to tickets currently made.

#### File Structure 
  - Add folder in assets for icons/svgs
    - addIcon
    - editIcon

#### Navbar
  - Create component LinkWrapper to clean up and simplify Navbar flow
  - Create NavLinks to clean up return from Navbar
  - Move links displayed based on access levels to variables rendered within NavLinks

#### createNewEvent.js
  - Small class changes

#### editProject.js
  - Remove All Projects button
  - CSS updates
  - Add Edit icon (redundent, ignore, merge conflicts resolved by Josh)

#### editableField.js
  - Refactor boolean ? \<component/> : null to:  boolean && \<component/>
  - Edit icon (ignore)
  - Add indication of { none } value

#### editableMeeting.js
  - Update CSS, classes

#### eventForm.js
  - Move section up in component flow as per UI spec

#### selectProject.js
  - add toggle state to switch between active tabs
  - Render AddNewProject in line with UI spec
    - This has errors, AddNewProject needs props from parent page component to function properly. I'm happy to figure out this factor so it works
  - There are still comments to be removed from this file

#### AddNewProject.js
  - Remove heading and link to dashboard to allow for clean render on selectProject.js

#### UserAdmin.js
  - Update initial page to PAGES.userManagement as per UI spec
    - I really dislike this route management pattern, I would like this to be refactored into sub routes later on.

#### variables.scss
  - Create scss file to house UI color pallete

#### index.scss
  - Add import for variables.scss

#### UserProfile.scss
  - Add import for variables.scss

#### CheckIn.scss
  - Add import for variables.scss
  - Multiple UI updates, move some color values to variable values for easier management

#### ManageProjects.scss
  - Add import for variables.scss
  - Multiple UI updates, move some color values to variable values for easier management
  - Additional classes for UI added here

#### Navbar.scss
  - Add import for variables.scss
  - Links
    - Change font size
    - Add active state
    - Update spacing

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals side by side wth changes</summary>

image

image

image

image

image

image

image

image

image

image

image

image

* After logout button
image

</details>

### CODE REVIEW

**Review by trillium — COMMENTED** [review]
Dangerously reviewing own commit

File: client/src/components/manageProjects/selectProject.js [hunk 1]
```diff
@@ -27,28 +32,53 @@ const SelectProject = ({ projects, accessLevel, user }) => {
...
+                width: 'auto',
+              }}
+              value='findProject'
+              onClick={handleToggle}
+            >
+              Find Project
+            </button>
+            {/* <Link to="useradmin"> */}
+
             <button
```
Comment (jbubar) [inline client/src/components/manageProjects/selectProject.js:53]: Somehow this button is broken. crashed the whole app
Screen Shot 2023-02-13 at 4 40 58 PM

**Review by jbubar — COMMENTED** [review]
Needs to be fixed.. looks good for the most part.. happy to work on this with you

**Comment by trillium** [comment]
Temporily resolved the crashing issue.

AddNewProject component needs props passed from UserAdmin page.

Suggest refactoring thoses props into a Context component that can be used by both Admin and User access levels.

**Comment by jbubar** [comment]
I fixed the merge conflicts... https://github.com/jbubar/VRMS/tree/Spiteless-ui_updates

**Comment by jbubar** [comment]
I cant push it because it is on trillium's fork

**Comment by trillium** [comment]
What needs to happen next?

**Review by jbubar — COMMENTED** [review]
I think this needs a lot of work still. but I am happy to merge it now.. I looked through the code and tested all of our functionality..
note: like trillium said, the create new project functionality on the project page still needs to be built out. 
Picture of the functionality that still needs to be built v v v
Screen Shot 2023-02-22 at 3 13 08 PM

**Comment by trillium** [comment]
I hear you on it needing a lot of work. I was trying to sprint prior to the Monday meeting to get all the ui changes in place so that the demo could be on the actual site instead of a figma file

**Comment by trillium** [comment]
I think it'd be nice for it to be merged in and then tweaked by the group to fit better

**Comment by MattPereira** [comment]
@Spiteless 

Please fill out the template that I pasted into the creation comment of this PR to help me better understand and review your changes

**Comment by trillium** [comment]
**Moved this comment to Pull Request heading

**Comment by trillium** [comment]
@MattPereira Requested update above

**Comment by MattPereira** [comment]
@Spiteless is this just a freestyle pull request or does it address a specific issue from the project board? 

I don't see you assigned to any of the issues from the [In Progress Column](https://github.com/hackforla/VRMS/projects/12#column-19074773) and you didnt replace the text in the template

```
Fixes #replace_this_text_with_the_issue_number
```

**Comment by trillium** [comment]
> @Spiteless is this just a freestyle pull request or does it address a specific issue from the project board?

**The content of the pull request has been updated. I thought you wanted it in a comment. Moved it to the top of the page**

It was originally me being overzealous and hoping that we'd have a live site that Bonnie could use to do the user testing that we were planning on last meeting. I feel a bit embarrased about this PR because it's very much not within the structure this project has been asking for. It does fix #1235, but it also goes way beyond it.

**Comment by MattPereira** [comment]
@Spiteless I admire your enthusiasm for sure

Apologies if my last response was a bit uncharitable. My intention was not to make you feel embarrassed. 

I've just spent the last few months training on the [hackforla/website](https://github.com/hackforla/website) team where one pull request is always linked to one issue so that reviewers have a blueprint for assessing if a PR is ready to be merged

Thanks for explaining the situation with this PR

**Comment by trillium** [comment]
I wanted to clarify, your comments were fine. I was embarrassed about this pull request being generated outside the scope of the projects current tickets. I only made this choice because I hadn't been able to contribute much over the last two months and we were short on contributors prior to last weeks meeting.

**Comment by jbubar** [comment]
I was thinking about merging it directly to Development using my git command line, but that would be bad practice, so I created a new pr for this. 

#1295 


@Spiteless can you review it to make sure my conflict resolution is right?
@MattPereira can you merge it after that?

**Comment by MattPereira** [comment]
@jbubar Help me understand why you are creating pull requests for @Spiteless ? 

If its his code shouldn't he be making the pull request? 

Also really confused why you closed this PR coming from Spiteless fork and opened a new PR from your own fork?

**Comment by trillium** [comment]
@MattPereira Yes this is being done wrong. Originally Josh opened the PR for me because I was working and we were trying to move quickly. Turns out it's not the best way to go about it. I'll open a PR from my branch.

**Pull request closed without merging**